packages/create-sitecore-jss/src/templates/react/src/lib/layout-service-factory.graphql.js

```diff
@@ -0,0 +1,14 @@
+import { GraphQLLayoutService } from '@sitecore-jss/sitecore-jss-react';
+import config from '../temp/config';
```

Comment:
As I understand, we don't have `jss-create` file anymore, and in this case we don't have file replacement. So, you should have a single factory which will use ejs templating, to use Graphql or Rest service